### PR TITLE
Fix Unit Tests

### PR DESCRIPTION
**Pull request by kchitalia**
All unit tests are now running and passing

### CODE REVIEW

File: SalesforceSDK/Salesforce.SDK.Tests/Package.appxmanifest [hunk 1]
```diff
@@ -5,40 +5,41 @@
   xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
   IgnorableNamespaces="uap mp">
 
-  <Identity Name="4f80cb65-3b76-48ce-a9f4-9c433fac03c5"
-            Publisher="CN=seattle"
+  <Identity Name="8a9e3bb8-91b4-452b-ae12-91207224ee23"
+            Publisher="CN=kchitalia"
```
Comment (bhariharan) [inline SalesforceSDK/Salesforce.SDK.Tests/Package.appxmanifest]: Shouldn't the publisher be something like ```Salesforce```?

File: SalesforceSDK/Salesforce.SDK.Tests/Package.appxmanifest [hunk 2]
```diff
@@ -5,40 +5,41 @@
...
             Version="1.0.0.0" />
 
-  <mp:PhoneIdentity PhoneProductId="4f80cb65-3b76-48ce-a9f4-9c433fac03c5" PhonePublisherId="00000000-0000-0000-0000-000000000000"/>
+  <mp:PhoneIdentity PhoneProductId="8a9e3bb8-91b4-452b-ae12-91207224ee23" PhonePublisherId="00000000-0000-0000-0000-000000000000"/>
 
   <Properties>
-    <DisplayName>UnitTestProject1</DisplayName>
-    <PublisherDisplayName>seattle</PublisherDisplayName>
+    <DisplayName>Tests</DisplayName>
+    <PublisherDisplayName>kchitalia</PublisherDisplayName>
```
Comment (bhariharan) [inline SalesforceSDK/Salesforce.SDK.Tests/Package.appxmanifest]: Same here for ```PublisherDisplayName```.

File: SalesforceSDK/Salesforce.SDK.Tests/Package.appxmanifest (hunk 1 above)
Comment (kchitalia) [inline SalesforceSDK/Salesforce.SDK.Tests/Package.appxmanifest]: Good catch :). Will change it.

File: SalesforceSDK/Salesforce.SDK.Tests/Package.appxmanifest (hunk 2 above)
Comment (kchitalia) [inline SalesforceSDK/Salesforce.SDK.Tests/Package.appxmanifest]: Will change it to Salesforce

**Comment by bhariharan** [comment]
Looks fine otherwise.

File: SalesforceSDK/Salesforce.SDK.Tests/Package.appxmanifest [hunk 3]
```diff
@@ -1,44 +1,28 @@
...
-  IgnorableNamespaces="uap mp">
-
-  <Identity Name="4f80cb65-3b76-48ce-a9f4-9c433fac03c5"
-            Publisher="CN=seattle"
-            Version="1.0.0.0" />
-
-  <mp:PhoneIdentity PhoneProductId="4f80cb65-3b76-48ce-a9f4-9c433fac03c5" PhonePublisherId="00000000-0000-0000-0000-000000000000"/>
-
+<Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10" xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest" xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10" IgnorableNamespaces="uap mp">
+  <Identity Name="8a9e3bb8-91b4-452b-ae12-91207224ee23" Publisher="CN=Salesforce" Version="1.0.0.0" />
```
Comment (kchitalia) [inline SalesforceSDK/Salesforce.SDK.Tests/Package.appxmanifest]: Changed to Salesforce.

File: SalesforceSDK/Salesforce.SDK.Tests/Package.appxmanifest [hunk 4]
```diff
@@ -1,44 +1,28 @@
...
+  <mp:PhoneIdentity PhoneProductId="8a9e3bb8-91b4-452b-ae12-91207224ee23" PhonePublisherId="00000000-0000-0000-0000-000000000000" />
   <Properties>
-    <DisplayName>UnitTestProject1</DisplayName>
-    <PublisherDisplayName>seattle</PublisherDisplayName>
+    <DisplayName>Tests</DisplayName>
+    <PublisherDisplayName>Salesforce</PublisherDisplayName>
```
Comment (kchitalia) [inline SalesforceSDK/Salesforce.SDK.Tests/Package.appxmanifest]: Changed to Salesforce

**Comment by bhariharan** [comment]
:+1:

**Pull request merged**